ui/Rhino/AGS/dev/AGS_toolbar_force_display_cmd.py

```diff
@@ -0,0 +1,57 @@
+from __future__ import print_function
+from __future__ import absolute_import
+from __future__ import division
+
+import scriptcontext as sc
+
+import compas_rhino
+
+import AGS_force_move_cmd
+import AGS_force_select_anchor_cmd
+import AGS_force_scale_cmd
+import AGS_force_displaysettings_cmd
+
+
+__commandname__ = "AGS_toolbar_force_display"
+
+
+def RunCommand(is_interactive):
+
+    if 'AGS' not in sc.sticky:
+        compas_rhino.display_message('AGS has not been initialised yet.')
+        return
+
+    scene = sc.sticky['AGS']['scene']
+    if not scene:
+        return
+
+    if not scene.find_by_name('Force'):
+        compas_rhino.display_message("There is no ForceDiagram in the scene.")
+        return
+
+    options = ["ForceLocation", "ForceAnchor", "ForceScale", "ForceDisplay"]
```

Comment:
anchor and location do not fit under display settings in my opinion

Reply:
I would suggest a "toolbar_cmd" `location` that takes care of `form location` an `force location`.

Force Ancho perhaps is a separated command.

Reply:
the anchor also exists for the form diagram (it also has a scale).

Reply:
Yep... you are correct... 
but I somehow feel anchor is a cmd that should be related to scale... (Maybe I delete these toolbar cmds for now.)

Reply:
i am all for deleting toolbar commands...

about the anchor and scale.

they should be independent because the anchor applies to more than just scale. in the general case any transformation can be applied to the "object" version of geometric data and/or a data structure.

however, i agree that, for convenience, some scaling commands can simultaneously modify the anchor position.